docs/articles_en/documentation/openvino-ir-format/operation-sets/operation-specs/movement/col2im-15.rst

```diff
@@ -0,0 +1,228 @@
+.. {#openvino_docs_ops_type_Col2Im_15}
+
+Col2Im
+===================
+
+
+.. meta::
+  :description: Learn about Col2Im-15 - data movement operation which combines sliding blocks into an image tensor.
+
+**Versioned name**: *Col2Im-15*
+
+**Category**: *Data movement*
+
+**Short description**: *Col2Im* operation constructs an image based on ``input`` tensor containing sliding data blocks (blocks of the image) and desired ``output_size``. 
+
+**Detailed description**
+
+Consider an ``input`` tensor containing batches of image blocks of shape ``(N, C * Product(kernel_size), L)``, where:
+
+* ``N`` is the batch dimension,
+* ``C * Product(kernel_size)`` is the number of elements within a block (each block contains ``Product(kernel_size)`` vectors containing values from each channel ``C``),
+* ``L`` is the total number of blocks calculated as follows:
+
+.. math::
+
+    L = \prod_{d=1}^{2} \lfloor \frac{({\text{{output\_size}}[d] + \text{{pads\_begin}}[d] + \text{{pads\_end}}[d] - \text{{dilation}}[d] \times (\text{{kernel\_size}}[d] - 1) - 1}}{{\text{{stride}}[d]}}) + 1\rfloor
```

Comment:
Small issue in built docs, backslash is not formatting how expected: 
![image](https://github.com/openvinotoolkit/openvino/assets/52485122/bccb8d80-16bb-4844-af2f-3d5e58a1d5bb)

Reply:
It's weird, since it's valid LaTeX syntax. I also tried using `\textunderscore`, but it doesn't render correctly either, and `output_size` is rendered with `size` as a subscript.

In the end I replaced the equation with its text representation, since I couldn't find an example of a variable name with `_` in other docs.